lib/db/schema/patch-014-013.sql

```diff
@@ -0,0 +1,60 @@
+-- -- drop the procedures we created
+-- DROP PROCEDURE `emailRecord_3`;
+-- DROP PROCEDURE `account_3`;
+
```

Comment:
As mentioned above, the only thing we'd have in the reverse patch are these two lines which drop the two newly added procedures. In theory these two lines should be in reverse order but for the purposes of this patch it doesn't matter. however it is good practice to do so. Cheers @eoger.